pyfstat/mcmc_based_searches.py

```diff
@@ -589,23 +486,28 @@ def run(
         Parameters
         ----------
         proposal_scale_factor: float
-            The proposal scale factor used by the sampler, see Goodman & Weare
-            (2010). If the acceptance fraction is too low, you can raise it by
-            decreasing the a parameter; and if it is too high, you can reduce
-            it by increasing the a parameter [Foreman-Mackay (2013)].
+            The proposal scale factor `a > 1` used by the sampler, see
+            [Goodman & Weare (2010) 10.2140/camcos.2010.5.65].
+            The bigger the value, the wider the range to draw proposals from.
+            If the acceptance fraction is too low, you can raise it by
+            decreasing the `a` parameter; and if it is too high, you can reduce
+            it by increasing the `a` parameter [Foreman-Mackay arXiv:1202.3665 [astro-ph.IM] (2013)].
```

Comment:
should also harmonize reference format here

Reply:
I haven't found any arXiv link, are you Ok keeping the doi? 93217b9c5198507488c3a4db05d02f996b1a2ce2

(I still have to update the Formean-Mackay one)

Reply:
:+1: